src/test/suite/extension.test.ts

```diff
@@ -260,8 +261,8 @@ suite(`Integration Tests`, () => {
         catch (error) {
             if (error instanceof Error) {
                 if (`response` in error) {
-                    // eslint-disable-next-line @typescript-eslint/dot-notation
-                    assert(error[`response`][`status`] === 404);
+                    const err = error as AxiosError;
+                    assert(err?.response?.status === 404);
```

Comment:
💡☕️ Just fyi - worth to check if nullable type is of any use in here, but if this is just for harness, I am all good, I noticed hence commenting. Thanks.

Reply:
Hi @Tatsinnit - I've reworked this to use the axios guard function which makes this a bit more readable. It's only there to test a 404